src/language-handlebars/printer-glimmer.js

```diff
@@ -356,7 +356,9 @@ function print(path, options, print) {
         trailingSpace = "";
       }
 
-      text = text.replace(/^\s+/g, leadingSpace).replace(/\s+$/, trailingSpace);
+      text = text
+        .replace(/^[\t\n\f\r ]+/g, leadingSpace)
```

Comment:
Let's see  if any test fails, if not, please add some test.

Reply:
I modified the git history to:
- add a test, I pushed this commit to the repo, it should ✅
- modify the printer to only replace html whitespaces (and not vertical tabulations), as a result CI should ❌
- record the tests, CI ✅